lte/gateway/c/oai/include/amf_app_messages_types.h

```diff
@@ -0,0 +1,87 @@
+
+
+/* copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
```

Comment:
Make the copyright consistent across all files

Reply:
okay